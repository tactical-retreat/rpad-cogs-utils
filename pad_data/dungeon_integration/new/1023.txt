Legelonte Descended! - Music Dragon-Legend

stage 0

495 - Love Deity, Feline Bastet @ level 2
Meow Meoooow
Absorb combo <= 3 damage for 99 turns
----- Turn 1 -----
Spread Cat
4857 Damage - Spawn random 5 Jammer orbs, Deal 150% damage (1 time use)
----- HP <= 30 -----
Cat Combination
6476 Damage - Deal 200% damage (4 hits, 50% each)

stage 1

1100 - Sea God's Songstress, Siren @ level 2
Abyssal Pressure
Player -99% HP
----- HP <= 50 -----
Voice of Misfortune
Bind Wood cards for 2~3 turns (1 time use)
----- Execute below actions in order repeatedly -----
Song of Destruction
884 Damage - Deal 50% damage
Song of Destruction
1769 Damage - Deal 100% damage
Song of Destruction
3538 Damage - Deal 200% damage
Song of Destruction
14152 Damage - Deal 800% damage

stage 2

1115 - Purple Lotus Chanter, Lakshmi @ level 2
Welcome! I'm glad you're here.
Player recover 100% hp
----- Turn 1 -----
Waterways Verification
Water skyfall +20% for 99 turns
----- HP <= 100 -----
Lotus Drop
5149 Damage - Change Heal to Water, Deal 150% damage (50% chance)
----- HP <= 70 -----
Chanchala
Leader changes to random sub for 3 turns (1 time use)
----- HP <= 50 -----
Magic Absorbing Eye
3432 Damage - Enemy recover 30% hp, Deal 100% damage
----- HP <= 20 -----
Wave Current Attack
8581 Damage - Deal 250% damage

stage 3

1917 - Hungry Peko Dragon @ level 2
I'm hungry...
Absorb Fire damage for 5 turns
----- HP <= 100 -----
I'm starving...Chomp!
2793 Damage - Change Fire to Jammer, Deal 150% damage (50% chance)
Mineral Throw
3910 Damage - Deal 210% damage (3 hits, 70% each)
----- HP <= 30 -----
Tiny Fire Breath
4655 Damage - Change 2nd, 3rd row to Fire orbs, Deal 250% damage

1918 - Battle Flower Spirit, Mandrake @ level 2
Strange Scream
Delay active skills by 3 turns
----- Turn 1 -----
Rock of Mandragora
4693 Damage - Deal 150% damage (5 hits, 30% each) (1 time use)
----- HP <= 100 -----
Gets excited
Do nothing
----- HP <= 30 -----
Mandrake Beam
6257 Damage - Change 3rd row to Wood orbs, Deal 200% damage

stage 4

1916 - Music Dragon, Legelonte @ level 2
The show begins
Voids status ailments for 999 turns
----- Execute below actions in order repeatedly -----
Piano
Bind 2 random subs for 3 turns
Forte
2296 Damage - Deal 50% damage
Pianissimo
Bind active skills for 2 turns
Fortissimo
4593 Damage - Deal 100% damage (2 hits, 50% each)
Pianississimo
Player -99% HP
Fortississimo
6890 Damage - Deal 150% damage (3 hits, 50% each)
Finale
13781 Damage - Deal 300% damage

stage 0

495 - Love Deity, Feline Bastet @ level 3
Meow Meoooow
Absorb combo <= 3 damage for 99 turns
----- Turn 1 -----
Spread Cat
8564 Damage - Spawn random 5 Jammer orbs, Deal 150% damage (1 time use)
----- HP <= 30 -----
Cat Combination
11419 Damage - Deal 200% damage (4 hits, 50% each)

stage 1

1100 - Sea God's Songstress, Siren @ level 3
Abyssal Pressure
Player -99% HP
----- HP <= 50 -----
Voice of Misfortune
Bind Wood cards for 2~3 turns (1 time use)
----- Execute below actions in order repeatedly -----
Song of Destruction
1559 Damage - Deal 50% damage
Song of Destruction
3119 Damage - Deal 100% damage
Song of Destruction
6238 Damage - Deal 200% damage
Song of Destruction
24953 Damage - Deal 800% damage

stage 2

1115 - Purple Lotus Chanter, Lakshmi @ level 3
Welcome! I'm glad you're here.
Player recover 100% hp
----- Turn 1 -----
Waterways Verification
Water skyfall +20% for 99 turns
----- HP <= 100 -----
Lotus Drop
9078 Damage - Change Heal to Water, Deal 150% damage (50% chance)
----- HP <= 70 -----
Chanchala
Leader changes to random sub for 3 turns (1 time use)
----- HP <= 50 -----
Magic Absorbing Eye
6052 Damage - Enemy recover 30% hp, Deal 100% damage
----- HP <= 20 -----
Wave Current Attack
15130 Damage - Deal 250% damage

stage 3

1917 - Hungry Peko Dragon @ level 3
I'm hungry...
Absorb Fire damage for 5 turns
----- HP <= 100 -----
I'm starving...Chomp!
4924 Damage - Change Fire to Jammer, Deal 150% damage (50% chance)
Mineral Throw
6894 Damage - Deal 210% damage (3 hits, 70% each)
----- HP <= 30 -----
Tiny Fire Breath
8207 Damage - Change 2nd, 3rd row to Fire orbs, Deal 250% damage

1918 - Battle Flower Spirit, Mandrake @ level 3
Strange Scream
Delay active skills by 3 turns
----- Turn 1 -----
Rock of Mandragora
8274 Damage - Deal 150% damage (5 hits, 30% each) (1 time use)
----- HP <= 100 -----
Gets excited
Do nothing
----- HP <= 30 -----
Mandrake Beam
11032 Damage - Change 3rd row to Wood orbs, Deal 200% damage

stage 4

1916 - Music Dragon, Legelonte @ level 4
The show begins
Voids status ailments for 999 turns
----- Execute below actions in order repeatedly -----
Piano
Bind 2 random subs for 3 turns
Forte
5802 Damage - Deal 50% damage
Pianissimo
Bind active skills for 2 turns
Fortissimo
11605 Damage - Deal 100% damage (2 hits, 50% each)
Pianississimo
Player -99% HP
Fortississimo
17407 Damage - Deal 150% damage (3 hits, 50% each)
Finale
34815 Damage - Deal 300% damage

Legelonte Descended! - Music Dragon-Master

stage 0

495 - Love Deity, Feline Bastet @ level 2
Meow Meoooow
Absorb combo <= 3 damage for 99 turns
----- Turn 1 -----
Spread Cat
4857 Damage - Spawn random 5 Jammer orbs, Deal 150% damage (1 time use)
----- HP <= 30 -----
Cat Combination
6476 Damage - Deal 200% damage (4 hits, 50% each)

stage 1

1100 - Sea God's Songstress, Siren @ level 2
Abyssal Pressure
Player -99% HP
----- HP <= 50 -----
Voice of Misfortune
Bind Wood cards for 2~3 turns (1 time use)
----- Execute below actions in order repeatedly -----
Song of Destruction
884 Damage - Deal 50% damage
Song of Destruction
1769 Damage - Deal 100% damage
Song of Destruction
3538 Damage - Deal 200% damage
Song of Destruction
14152 Damage - Deal 800% damage

stage 2

1115 - Purple Lotus Chanter, Lakshmi @ level 2
Welcome! I'm glad you're here.
Player recover 100% hp
----- Turn 1 -----
Waterways Verification
Water skyfall +20% for 99 turns
----- HP <= 100 -----
Lotus Drop
5149 Damage - Change Heal to Water, Deal 150% damage (50% chance)
----- HP <= 70 -----
Chanchala
Leader changes to random sub for 3 turns (1 time use)
----- HP <= 50 -----
Magic Absorbing Eye
3432 Damage - Enemy recover 30% hp, Deal 100% damage
----- HP <= 20 -----
Wave Current Attack
8581 Damage - Deal 250% damage

stage 3

1917 - Hungry Peko Dragon @ level 2
I'm hungry...
Absorb Fire damage for 5 turns
----- HP <= 100 -----
I'm starving...Chomp!
2793 Damage - Change Fire to Jammer, Deal 150% damage (50% chance)
Mineral Throw
3910 Damage - Deal 210% damage (3 hits, 70% each)
----- HP <= 30 -----
Tiny Fire Breath
4655 Damage - Change 2nd, 3rd row to Fire orbs, Deal 250% damage

1918 - Battle Flower Spirit, Mandrake @ level 2
Strange Scream
Delay active skills by 3 turns
----- Turn 1 -----
Rock of Mandragora
4693 Damage - Deal 150% damage (5 hits, 30% each) (1 time use)
----- HP <= 100 -----
Gets excited
Do nothing
----- HP <= 30 -----
Mandrake Beam
6257 Damage - Change 3rd row to Wood orbs, Deal 200% damage

stage 4

1916 - Music Dragon, Legelonte @ level 2
The show begins
Voids status ailments for 999 turns
----- Execute below actions in order repeatedly -----
Piano
Bind 2 random subs for 3 turns
Forte
2296 Damage - Deal 50% damage
Pianissimo
Bind active skills for 2 turns
Fortissimo
4593 Damage - Deal 100% damage (2 hits, 50% each)
Pianississimo
Player -99% HP
Fortississimo
6890 Damage - Deal 150% damage (3 hits, 50% each)
Finale
13781 Damage - Deal 300% damage

stage 0

495 - Love Deity, Feline Bastet @ level 3
Meow Meoooow
Absorb combo <= 3 damage for 99 turns
----- Turn 1 -----
Spread Cat
8564 Damage - Spawn random 5 Jammer orbs, Deal 150% damage (1 time use)
----- HP <= 30 -----
Cat Combination
11419 Damage - Deal 200% damage (4 hits, 50% each)

stage 1

1100 - Sea God's Songstress, Siren @ level 3
Abyssal Pressure
Player -99% HP
----- HP <= 50 -----
Voice of Misfortune
Bind Wood cards for 2~3 turns (1 time use)
----- Execute below actions in order repeatedly -----
Song of Destruction
1559 Damage - Deal 50% damage
Song of Destruction
3119 Damage - Deal 100% damage
Song of Destruction
6238 Damage - Deal 200% damage
Song of Destruction
24953 Damage - Deal 800% damage

stage 2

1115 - Purple Lotus Chanter, Lakshmi @ level 3
Welcome! I'm glad you're here.
Player recover 100% hp
----- Turn 1 -----
Waterways Verification
Water skyfall +20% for 99 turns
----- HP <= 100 -----
Lotus Drop
9078 Damage - Change Heal to Water, Deal 150% damage (50% chance)
----- HP <= 70 -----
Chanchala
Leader changes to random sub for 3 turns (1 time use)
----- HP <= 50 -----
Magic Absorbing Eye
6052 Damage - Enemy recover 30% hp, Deal 100% damage
----- HP <= 20 -----
Wave Current Attack
15130 Damage - Deal 250% damage

stage 3

1917 - Hungry Peko Dragon @ level 3
I'm hungry...
Absorb Fire damage for 5 turns
----- HP <= 100 -----
I'm starving...Chomp!
4924 Damage - Change Fire to Jammer, Deal 150% damage (50% chance)
Mineral Throw
6894 Damage - Deal 210% damage (3 hits, 70% each)
----- HP <= 30 -----
Tiny Fire Breath
8207 Damage - Change 2nd, 3rd row to Fire orbs, Deal 250% damage

1918 - Battle Flower Spirit, Mandrake @ level 3
Strange Scream
Delay active skills by 3 turns
----- Turn 1 -----
Rock of Mandragora
8274 Damage - Deal 150% damage (5 hits, 30% each) (1 time use)
----- HP <= 100 -----
Gets excited
Do nothing
----- HP <= 30 -----
Mandrake Beam
11032 Damage - Change 3rd row to Wood orbs, Deal 200% damage

stage 4

1916 - Music Dragon, Legelonte @ level 4
The show begins
Voids status ailments for 999 turns
----- Execute below actions in order repeatedly -----
Piano
Bind 2 random subs for 3 turns
Forte
5802 Damage - Deal 50% damage
Pianissimo
Bind active skills for 2 turns
Fortissimo
11605 Damage - Deal 100% damage (2 hits, 50% each)
Pianississimo
Player -99% HP
Fortississimo
17407 Damage - Deal 150% damage (3 hits, 50% each)
Finale
34815 Damage - Deal 300% damage

