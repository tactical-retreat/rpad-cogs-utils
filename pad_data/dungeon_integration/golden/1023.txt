Legelonte Descended! - Music Dragon-Legend

stage 0

495 - Love Deity, Feline Bastet @ level 2
Meow Meoooow
Absorb combo <= 3 damage for 99 turns
----- Turn 1 -----
Spread Cat
4857 Damage - Spawn random 5 Jammer orbs, Deal 150% damage
----- HP <= 30 -----
Cat Combination
6476 Damage - Deal 200% damage (4 hits, 50% each)

stage 1

1100 - Sea God's Songstress, Siren @ level 2
Abyssal Pressure
Player -99% HP
----- HP <= 50 -----
Voice of Misfortune
Bind Wood cards for 2~3 turns

stage 2

1115 - Purple Lotus Chanter, Lakshmi @ level 2
Welcome! I'm glad you're here.
Player recover 100% hp
----- Turn 1 -----
Waterways Verification
Water skyfall +20% for 99 turns
----- HP <= 100 -----
Lotus Drop
5149 Damage - Change Heal to Water, Deal 150% damage
----- HP <= 70 -----
Chanchala
Leader changes to random sub for 3 turns
----- HP <= 50 -----
Magic Absorbing Eye
3432 Damage - Enemy recover 30% hp, Deal 100% damage
----- HP <= 20 -----
Wave Current Attack
8581 Damage - Deal 250% damage

stage 3

1917 - Hungry Peko Dragon @ level 2
I'm hungry...
Absorb Fire damage for 5 turns
----- HP <= 100 -----
I'm starving...Chomp!
2793 Damage - Change Fire to Jammer, Deal 150% damage
Mineral Throw
3910 Damage - Deal 210% damage (3 hits, 70% each)
----- HP <= 30 -----
Tiny Fire Breath
4655 Damage - Change 2nd, 3rd row to Fire orbs, Deal 250% damage

1918 - Battle Flower Spirit, Mandrake @ level 2
Strange Scream
Delay active skills by 3 turns
----- Turn 1 -----
Rock of Mandragora
4693 Damage - Deal 150% damage (5 hits, 30% each)
----- HP <= 100 -----
Gets excited
Do nothing
----- HP <= 30 -----
Mandrake Beam
6257 Damage - Change 3rd row to Wood orbs, Deal 200% damage

stage 4

1916 - Music Dragon, Legelonte @ level 2
The show begins
Voids status ailments for 999 turns

stage 0

495 - Love Deity, Feline Bastet @ level 3
Meow Meoooow
Absorb combo <= 3 damage for 99 turns
----- Turn 1 -----
Spread Cat
8564 Damage - Spawn random 5 Jammer orbs, Deal 150% damage
----- HP <= 30 -----
Cat Combination
11419 Damage - Deal 200% damage (4 hits, 50% each)

stage 1

1100 - Sea God's Songstress, Siren @ level 3
Abyssal Pressure
Player -99% HP
----- HP <= 50 -----
Voice of Misfortune
Bind Wood cards for 2~3 turns

stage 2

1115 - Purple Lotus Chanter, Lakshmi @ level 3
Welcome! I'm glad you're here.
Player recover 100% hp
----- Turn 1 -----
Waterways Verification
Water skyfall +20% for 99 turns
----- HP <= 100 -----
Lotus Drop
9078 Damage - Change Heal to Water, Deal 150% damage
----- HP <= 70 -----
Chanchala
Leader changes to random sub for 3 turns
----- HP <= 50 -----
Magic Absorbing Eye
6052 Damage - Enemy recover 30% hp, Deal 100% damage
----- HP <= 20 -----
Wave Current Attack
15130 Damage - Deal 250% damage

stage 3

1917 - Hungry Peko Dragon @ level 3
I'm hungry...
Absorb Fire damage for 5 turns
----- HP <= 100 -----
I'm starving...Chomp!
4924 Damage - Change Fire to Jammer, Deal 150% damage
Mineral Throw
6894 Damage - Deal 210% damage (3 hits, 70% each)
----- HP <= 30 -----
Tiny Fire Breath
8207 Damage - Change 2nd, 3rd row to Fire orbs, Deal 250% damage

1918 - Battle Flower Spirit, Mandrake @ level 3
Strange Scream
Delay active skills by 3 turns
----- Turn 1 -----
Rock of Mandragora
8274 Damage - Deal 150% damage (5 hits, 30% each)
----- HP <= 100 -----
Gets excited
Do nothing
----- HP <= 30 -----
Mandrake Beam
11032 Damage - Change 3rd row to Wood orbs, Deal 200% damage

stage 4

1916 - Music Dragon, Legelonte @ level 4
The show begins
Voids status ailments for 999 turns

Legelonte Descended! - Music Dragon-Master

stage 0

495 - Love Deity, Feline Bastet @ level 2
Meow Meoooow
Absorb combo <= 3 damage for 99 turns
----- Turn 1 -----
Spread Cat
4857 Damage - Spawn random 5 Jammer orbs, Deal 150% damage
----- HP <= 30 -----
Cat Combination
6476 Damage - Deal 200% damage (4 hits, 50% each)

stage 1

1100 - Sea God's Songstress, Siren @ level 2
Abyssal Pressure
Player -99% HP
----- HP <= 50 -----
Voice of Misfortune
Bind Wood cards for 2~3 turns

stage 2

1115 - Purple Lotus Chanter, Lakshmi @ level 2
Welcome! I'm glad you're here.
Player recover 100% hp
----- Turn 1 -----
Waterways Verification
Water skyfall +20% for 99 turns
----- HP <= 100 -----
Lotus Drop
5149 Damage - Change Heal to Water, Deal 150% damage
----- HP <= 70 -----
Chanchala
Leader changes to random sub for 3 turns
----- HP <= 50 -----
Magic Absorbing Eye
3432 Damage - Enemy recover 30% hp, Deal 100% damage
----- HP <= 20 -----
Wave Current Attack
8581 Damage - Deal 250% damage

stage 3

1917 - Hungry Peko Dragon @ level 2
I'm hungry...
Absorb Fire damage for 5 turns
----- HP <= 100 -----
I'm starving...Chomp!
2793 Damage - Change Fire to Jammer, Deal 150% damage
Mineral Throw
3910 Damage - Deal 210% damage (3 hits, 70% each)
----- HP <= 30 -----
Tiny Fire Breath
4655 Damage - Change 2nd, 3rd row to Fire orbs, Deal 250% damage

1918 - Battle Flower Spirit, Mandrake @ level 2
Strange Scream
Delay active skills by 3 turns
----- Turn 1 -----
Rock of Mandragora
4693 Damage - Deal 150% damage (5 hits, 30% each)
----- HP <= 100 -----
Gets excited
Do nothing
----- HP <= 30 -----
Mandrake Beam
6257 Damage - Change 3rd row to Wood orbs, Deal 200% damage

stage 4

1916 - Music Dragon, Legelonte @ level 2
The show begins
Voids status ailments for 999 turns

stage 0

495 - Love Deity, Feline Bastet @ level 3
Meow Meoooow
Absorb combo <= 3 damage for 99 turns
----- Turn 1 -----
Spread Cat
8564 Damage - Spawn random 5 Jammer orbs, Deal 150% damage
----- HP <= 30 -----
Cat Combination
11419 Damage - Deal 200% damage (4 hits, 50% each)

stage 1

1100 - Sea God's Songstress, Siren @ level 3
Abyssal Pressure
Player -99% HP
----- HP <= 50 -----
Voice of Misfortune
Bind Wood cards for 2~3 turns

stage 2

1115 - Purple Lotus Chanter, Lakshmi @ level 3
Welcome! I'm glad you're here.
Player recover 100% hp
----- Turn 1 -----
Waterways Verification
Water skyfall +20% for 99 turns
----- HP <= 100 -----
Lotus Drop
9078 Damage - Change Heal to Water, Deal 150% damage
----- HP <= 70 -----
Chanchala
Leader changes to random sub for 3 turns
----- HP <= 50 -----
Magic Absorbing Eye
6052 Damage - Enemy recover 30% hp, Deal 100% damage
----- HP <= 20 -----
Wave Current Attack
15130 Damage - Deal 250% damage

stage 3

1917 - Hungry Peko Dragon @ level 3
I'm hungry...
Absorb Fire damage for 5 turns
----- HP <= 100 -----
I'm starving...Chomp!
4924 Damage - Change Fire to Jammer, Deal 150% damage
Mineral Throw
6894 Damage - Deal 210% damage (3 hits, 70% each)
----- HP <= 30 -----
Tiny Fire Breath
8207 Damage - Change 2nd, 3rd row to Fire orbs, Deal 250% damage

1918 - Battle Flower Spirit, Mandrake @ level 3
Strange Scream
Delay active skills by 3 turns
----- Turn 1 -----
Rock of Mandragora
8274 Damage - Deal 150% damage (5 hits, 30% each)
----- HP <= 100 -----
Gets excited
Do nothing
----- HP <= 30 -----
Mandrake Beam
11032 Damage - Change 3rd row to Wood orbs, Deal 200% damage

stage 4

1916 - Music Dragon, Legelonte @ level 4
The show begins
Voids status ailments for 999 turns

